packages/framework/tinylicious-client/package.json

```diff
@@ -41,20 +41,20 @@
   "dependencies": {
```

Comment:
Please update the peerDependencies "fluid-framework": "^2.0.0"

I can't comment directly on the line in github.